applications/vessel-history/src/features/map/map.context.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {
```

Comment:
I realised a couple of few days ago the react-map-gl library already offers a context component to read map properties out of the map.
I'd use it here instead of the custom implementation we did, ref: https://github.com/GlobalFishingWatch/frontend/blob/master/applications/fishing-map/src/features/map/map-context.hooks.ts